src/runtime/plugin.ts

```diff
@@ -7,6 +7,6 @@ export default defineNuxtPlugin(async (nuxtApp) => {
 
   nuxtApp.vueApp.use(clerkPlugin, {
     publishableKey,
-    options: NuxtClerkModule.vueClerkOptions,
+    ...NuxtClerkModule.vueClerkOptions,
```

Comment:
`options` property is removed in favor of top-level options. See https://github.com/wobsoriano/vue-clerk/pull/26